python/src/lib.rs

```diff
@@ -234,5 +283,9 @@ fn dora_ros2_bridge(_py: Python, m: &PyModule) -> PyResult<()> {
     m.add_class::<qos::Ros2QosPolicies>()?;
     m.add_class::<qos::Ros2Durability>()?;
     m.add_class::<qos::Ros2Liveliness>()?;
+
+    m.add_function(wrap_pyfunction!(dora::start_runtime, m)?)?;
+    m.add_class::<Node>().unwrap();
```

Comment:
But happy to discuss possibilities!

Reply:
Yes, I was also thinking about which option is better. One of the libraries has to depend on the other.

The best solution, in my opinion, would be to add an optional `ros2-bridge` feature to the main `dora` project, which then imports the bridge and provides the `create_subscription_stream` method. Unfortunately, such an optional feature is not possible for Python libraries since they're precompiled. So this would mean that the `dora` API libraries always include the ROS2 bridge functionality, so it would become a first-class feature instead of a separate extension crate.

I wasn't sure whether we want to do this, so I chose to make the bridge depend on `dora` instead. I'm happy to change it if you prefer it the other way around.

Reply:
Including the bridge in the main dora project would also simplify the setup considerably, as we only need to use `maturin` in one place. Also, we cut the bridge in the same repo, which makes version management much easier.

Reply:
Ultimately we can merge this PR as it is so that we can merge all currently PR before migrating everything. If that sounds good to you

Reply:
Sounds good to me!